slick/src/main/scala/slick/jdbc/JdbcActionComponent.scala

```diff
@@ -666,7 +666,7 @@ trait JdbcActionComponent extends SqlActionComponent { self: JdbcProfile =>
     protected def retMany(values: Iterable[U], individual: Seq[SingleInsertResult]) = individual
 
     protected def retManyBatch(st: Statement, values: Iterable[U], updateCounts: Array[Int]) =
```

Comment:
Binary compatibilty issue, the return type of this method changed. You can fix this by adding `: Seq[RU]` as the return type of this method

Reply:
I made this change just now